### PR TITLE
Added homework_18_310123

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Could you check my homework18 (HashMap, TreeMap) ?

Thanks in advance!

### CODE REVIEW

File: homework_18_310123/src/TreeMapMethods.java [hunk 1]
```diff
@@ -0,0 +1,379 @@
+import java.util.*;
+
+public class TreeMapMethods {
+    //    Java Collection: TreeMap Exercises [26 exercises with solution]
```
Comment (YeaaaaH) [inline homework_18_310123/src/TreeMapMethods.java:4]: ??

File: homework_18_310123/src/HashMapMethods.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class HashMapMethods {
+
+    //    Java Collection: HashMap Exercises [12 exercises with solution]
```
Comment (YeaaaaH) [inline homework_18_310123/src/HashMapMethods.java:7]: ??

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
remove it)

File: homework_18_310123/src/TreeMapMethods.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_18_310123/src/TreeMapMethods.java:4]: Fixed.

File: homework_18_310123/src/HashMapMethods.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_18_310123/src/HashMapMethods.java:7]: Fixed

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**